src/main/java/racingcar/service/RaceService.java

```diff
@@ -0,0 +1,31 @@
+package racingcar.service;
+
+import java.util.List;
+import java.util.Map;
+
+public class RaceService {
+
+    private final CarManager carManager = new CarManager();
+    private final Race race = new Race();
+    private final WinnerFinder winnerFinder = new WinnerFinder();
+    private int attemptCount;
+    private List<String> carNames;
+
+    public RaceService(List<String> carNames, int attemptCount) {
+        this.attemptCount = attemptCount;
+        this.carNames = carNames;
+    }
+
+    public void start() {
+        carManager.createCars(carNames);
+        race.run(attemptCount, carManager.getCars());
+    }
```

Comment:
carNames와, attemptCount는 start()에서만 사용되는 것 같은데, 멤버 변수로 유지하는 것 대신 start 호출 시 인자로 넘겨주면서 사용하는 것은 어떨까요?

```suggestion
    public void start(List<String> carNames, int attemptCount) {
        carManager.createCars(carNames);
        race.run(attemptCount, carManager.getCars());
    }
```

Reply:
좋은 방향인것 같아요